src/components/Actions.tsx

```diff
@@ -223,11 +224,11 @@ export const ShapesSwitcher = ({
   app: AppClassProperties;
 }) => {
   const [isExtraToolsMenuOpen, setIsExtraToolsMenuOpen] = useState(false);
-  const device = useDevice();
 
   const frameToolSelected = activeTool.type === "frame";
   const laserToolSelected = activeTool.type === "laser";
   const embeddableToolSelected = activeTool.type === "embeddable";
+  const mermaidToolSelected = appState.openDialog === "mermaid";
```

Comment:
we don't wanna highlight the more toools button or the mermaid button since it's a dialog and highlighting would serve no purpose as it'd be highlighted only while the dialog is open.